app/src/androidTest/java/com/duckduckgo/app/statistics/VariantManagerTest.kt

```diff
@@ -58,6 +58,29 @@ class VariantManagerTest {
         assertEqualsDouble(1.0, variant.weight)
         assertEquals(1, variant.features.size)
         assertTrue(variant.hasFeature(BottomBarNavigation))
+
+    }
+
+    // SERP Header Experiment
```

Comment:
Can we call these something else e.g "Single search bar experiments" to avoid confusion? When we refer to "SERP experiments" in mobile we're generally referring to SERP webpage changes that don't impact in-app code.